src/stores/index.js

```diff
@@ -2386,3 +2472,27 @@ export const notificationStore = rootStore.notificationStore;
 
 window.rootStore = rootStore;
 
+// Decode a signed JSON message
```

Comment:
for record - signing code will be moved to client
https://github.com/eluv-io/elv-client-js/pull/267#event-12093183969